### PR TITLE
obs-studio: Update to version 30.1

### LINKED ISSUE #125: obs-studio: the download url/filename has changed
obs-studio: couldn't match 'OBS-Studio-([\d.]+)-Full-x64\.zip' in https://obsproject.com/download

the regex to find the version in the file name seems to be something like "OBS-Studio-([\d.]+)\.zip" 
and the download url should reflect this "...\OBS-Studio-$version.zip"

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [ ] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
Update the checkver and the download url since the name changed
## Related Tickets & Documents
Fixes #125
## [optional] What gif/emoji best describes this PR or how it makes you feel?
:alien: :bug:
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### File persistence to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### obs-studio

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
